korge/src/commonMain/kotlin/com/soywiz/korge/newui/UIComboBox.kt

```diff
@@ -14,6 +15,7 @@ inline fun <T> Container.uiComboBox(
 	block: UIComboBox<T>.() -> Unit = {}
 ) = UIComboBox(width.toDouble(), height.toDouble(), selectedIndex, items, skin).also { addChild(it) }.also(block)
 
+@Deprecated("Use .ui.UIComboBox instead")
```

Comment:
Could it be a typealias with Deprecated? If not, since newui is pretty new, maybe we could just delete these clases

Reply:
I don't know about typealiases since there may be compile time errors after updating the korge dependency, if there are developers who used the old api in their projects.

Reply:
But if there are no such developers, I would delete the old classes too.